bot.js

```diff
@@ -49,11 +54,10 @@ client.on('message', message => {
 	}
 
 	if (message.content === `${prefix}lt3` || message.content === `${prefix}roster`) {
+		// probably should just remove the `${prefix}lt3` above or change it to
+		// import from ./guilds/{filename} strip the .json for porability and customization for other guilds
 		let roster = getRoster()
-		let rosterMsg = ""
-		roster = roster.map(member => member.name).sort()
-		roster.forEach(member => rosterMsg += member + " " )
-		message.channel.send(`${rosterMsg} (${roster.length})`)
+		postRoster.postRoster(message, roster);
```

Comment:
If we are going to extract all the commands into their own objects, what do you think about the paradigm of them all having a 'send' command, or 'render'? So that you would see:
```
postRoster.render(message, roster) 
find.render(message, roster)
demote.render(message, roster, leaders, name);
```

Reply:
breaking things down is all fine. the decision is up to the tech lead and you. im literally just moving around code into their own files. Not really adding or changing anything at all unless i encounter a little bug.

Reply:
Sounds like a job for @tommymurphy (Big T)

Reply:
@driedyam Tried this at one point and had issues.. but worth another shot